docs/modules/reference/pages/provisioning/adapters/snmp-metadata.adoc

```diff
@@ -28,6 +28,10 @@ The example above will query nodes with System Object Ids matching the regular e
 The data will be searched in the SNMP sub-tree `.1.2.3.4` and be stored in the node's metadata context `snmp`.
 The top-level element will be named `myHardware`.
 
+The provisioning adapter automatically appends the `.0` value to single entry OIDs per normal SNMP specification.
+There are edge cases where we will need to specify the entire OID statically to retrieve a single piece of data.
```

Comment:
This sentence sounds like a note to OpenNMS. Could we change to:

In some edge cases, you will need to specify the entire OID statically to retrieve a single piece of data. 

Can we specify what those edge cases might be, or how someone can determine when they would need to specify the entire OID, or will an error message appear if they do not?

Reply:
A SNMP scalar value has a .0 at the end of its OId, e.g. ".1.2.3.4.5.0". A list of values has its instance identifier at the end, e.g. ".1.2.3.4.5.1.2.3" where "1.2.3" is the instance id.

Reply:
Thanks for clarifying.